Symfony/CS/Fixer/Symfony/SpacesAfterSemicolonFixer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+class SpacesAfterSemicolonFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        for ($index = count($tokens) - 3; $index > 0; --$index) {
+            if (!$tokens[$index]->equals(';') || $tokens[$index + 2]->isComment()) {
+                continue;
+            }
+
+            if (!$tokens[$index + 1]->isWhitespace()) {
+                $tokens->insertAt($index + 1, new Token(array(T_WHITESPACE, ' ')));
+                continue;
+            }
+
+            if (' ' === $tokens[$index + 1]->getContent()) {
```

Comment:
Update the PR.
The `ensureWhitespaceAtIndex` is not going to work, since it does not check that when there is a whitespace the content is of the given one (one space in this case). So here it would not remove the whitespace that is to much :(